reachm-plus.yml

```diff
@@ -83,3 +83,23 @@ nav:
   - Integration with external software:
     - ArduPilot integration: ardupilot-integration.md
     - Bluetooth output and Android mock location: common/tutorials/mock-location.md
+model: M+
+power_port: JST-GH
+index_images:
+  - 'img/reachm-plus/quickstart.png'
+  - 'img/reachm-plus/tutorials.png'
+  - 'img/reachm-plus/reachview.png'
+  - 'img/reachm-plus/antenna-placement.png'
+  - 'img/reachm-plus/power-supply.png'
+  - 'img/reachm-plus/specs.png'
+kit:
+  - "* 1 x Reach M+ module"
```

Comment:
Here are double quotes as well. I believe we should use the same format everywhere

Reply:
Fixed including previously existing inconsistencies.